api/swagger.yaml

```diff
@@ -2790,6 +2790,24 @@ definitions:
           type: "array"
           items:
             $ref: "#/definitions/Mount"
+
+  Driver:
+    description: "Driver represents a driver (network, logging, secrets)."
+    type: "object"
+    required: [Name, Data]
```

Comment:
janky failure looks related. 

``` 
09:58:42 - The swagger spec at "api/swagger.yaml" is invalid against swagger specification 2.0. see errors :
09:58:42 - "Data" is present in required but not defined as property in definition "Driver"
09:58:42 - /secrets/{id}.Spec.Driver.Data in body is required
```

Q: I haven't looked at the swagger.yaml before, but I'm guessing Data is supposed to Options here, which is based off https://github.com/moby/moby/blob/0304c98d85404fe75a1b4a35d3c111931e062f41/api/types/swarm/common.go#L23 right?